packages/mdctl-core/streams/section.js

```diff
@@ -298,6 +310,55 @@ class ExportSection {
     }
   }
 
+  extractTemplatesGrouped() {
```

Comment:
if extractTemplates is not longer used please remove or use the same method name.